docs/examples/job-mysql.yaml

```diff
@@ -0,0 +1,81 @@
+# Copyright 2023 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+###
+# This demonstrates how to configure a batch job so that it shuts down
+# the proxy containers when it has finished processing.
+#
+# The operator will set an environment variable called CSQL_QUIT_URLS
+# with a space-separated list of URLs to each proxy container's /quitquitquit
+# endpoint. In most cases, your job will only have one proxy container attached,
+# so the value of CSQL_QUIT_URLS will be something like
+# "http://localhost:9091/quitquitquit"
+#
+# The main job container should send a POST request each URL when the job
```

Comment:
POST or GET? GET is so much more convenient.

Reply:
Also "request TO each URL"

Reply:
Fixed.